packages/sharing-smart-contract/contracts/ProtectedDataSharing.sol

```diff
@@ -298,15 +300,21 @@ contract ProtectedDataSharing is
         appForProtectedData[_collectionId][_protectedData] = _appAddress;
         protectedDataRegistry.safeTransferFrom(msg.sender, address(this), tokenId);
         protectedDatas[_collectionId][uint160(_protectedData)] = _protectedData;
+        protectedDataNumbersForCollection[_collectionId] += 1;
```

Comment:
Shouldn't this counter be also modified when removing a protected data from collection?

Reply:
yeah my bad i forgot ! Thanks

Reply:
Fix